Pyboard-D Latest Build - ordinal not in range(128)

ubuntu 18.04LTS

```
make -j6 BOARD=PYBD_SF2
Use make V=1 or set BUILD_VERBOSE in your environment to increase build verbosity.
make: Warning: File 'mpconfigport.mk' has modification time 0.32 s in the future
GEN build-PYBD_SF2/genhdr/moduledefs.h
../../py/py.mk:346: recipe for target 'build-PYBD_SF2/genhdr/moduledefs.h' failed
Traceback (most recent call last):
  File "../../py/makemoduledefs.py", line 107, in <module>
    main()
  File "../../py/makemoduledefs.py", line 101, in main
    modules |= find_module_registrations(c_file)
  File "../../py/makemoduledefs.py", line 53, in find_module_registrations
    return set(re.findall(pattern, c_file_obj.read()))
  File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 10887: ordinal not in range(128)
make: *** [build-PYBD_SF2/genhdr/moduledefs.h] Error 1
make: *** Deleting file 'build-PYBD_SF2/genhdr/moduledefs.h'
```

--- a/py/makemoduledefs.py
+++ b/py/makemoduledefs.py
@@ -8,6 +8,7 @@
 from __future__ import print_function
 
 import re
+import io
 import os
 import argparse
 
@@ -49,7 +50,7 @@ def find_module_registrations(c_file):
         # No c file to match the object file, skip
         return set()
 
-    with open(c_file) as c_file_obj:
+    with io.open(c_file, encoding='utf-8') as c_file_obj:
         return set(re.findall(pattern, c_file_obj.read()))
 
 

--- a/py/makeqstrdefs.py
+++ b/py/makeqstrdefs.py
@@ -9,6 +9,7 @@ from __future__ import print_function
 
 import re
 import sys
+import io
 import os
 
 # Blacklist of qstrings that are specially handled in further
@@ -108,7 +109,7 @@ if __name__ == "__main__":
         pass
 
     if args.command == "split":
-        with open(args.input_filename) as infile:
+        with io.open(args.input_filename, encoding='utf-8') as infile:
             process_file(infile)
 
     if args.command == "cat":

